__tests__/endpoints.test.js

```diff
@@ -0,0 +1,31 @@
+const request = require("supertest");
+const app = require("../app");
+
+describe("/api", () => {
```

Comment:
Requiring in endpoints.json in the test and comparing it to the content of the response would do, but I like that you make sure all the endpoint descriptions are there, but in this case, every time you complete a new endpoint, you'll need to write a test here as well to check if it's been added to endpoints.json.